src/i18n/es/nav.ts

```diff
@@ -29,8 +29,9 @@ export default NavDictionary({
 	'guides/rss': 'RSS',
 	'guides/styling': 'Estilos & CSS',
 	'guides/troubleshooting': 'Solución de problemas',
-	'reference/adapter-reference': 'API de adaptadores',
+	'reference/adapter-reference': 'API del adaptadores',
```

Comment:
Changing "de" for "del" does not make sense because "adaptadores" is a plural noun and "del" is used for singular nouns

Reply:
fix on #1214